src/components/matrix/grid-button.tsx

```diff
@@ -104,7 +105,8 @@ export const GridRightButton = ({ onClick }: GridMoveButton): JSX.Element => {
         height: 45,
```

Comment:
You should not hardcode the height and instead should be `columnHeaderHeight`. Similar to how the `GridUpButton` and `GridDownButton`'s width is `rowHeaderWidth`.

This will make sure the icon is vertically aligned in the middle.